images_to_samples.py

```diff
@@ -33,9 +33,10 @@ def mask_image(arrayA, arrayB):
     >>> x2 = np.array([1.5, 1.2, 1.6, 1.2, 11., 1.1, 25.9, 0.1], dtype=np.float32).reshape(2,2,2)
     >>> mask_image(x1, x2)
     array([[[ 0. ,  0. ],
-        [ 1.6,  1.2]],
-        [[11. ,  1.1],
-        [25.9,  0.1]]], dtype=float32)
+            [ 1.6,  1.2]],
```

Comment:
The result in the docstring is not coherent with what the code does. Can anyone confirm that the actual output (which I fixed here) is what we want from this function?

Reply:
I didn't test this docstring... who added this? @mpelchat04 do you know? This function was implemented before I arrived at CCCOT

Reply:
Yup, my bad.